def test_fail():
    name = 'taro'
    assert name == 1, f'nameのテストに失敗しました。name: {name}'
